### ISSUE TITLE
Diff deletion lines are not announced by screenreaders

### ISSUE THREAD

**Issue by outofambit** [desktop/desktop#6579]
"+" with Green background is read by NVDA, but the "-" with Red background is not - NVDA users will not understand what text has been removed. Visually +/- is great, but screenreaders should announce more "addition" or "deletion."

Control does not provide role, state, and value information.

**Comment by j-f1** [comment]
It might be because it’s reading the `-` as a hyphen and not pronouncing it.

**Comment by tidy-dev** [comment]
With recent changes, the + and - are now in their own elements and are announced. In addition the controls also provide added/deleted information. The line numbers also have screenreader only text that announce "deleted" or "added" when navigating the rows/columns with NVDA.
